jbpm-flow/src/main/java/org/jbpm/process/instance/AbstractProcessInstanceFactory.java

```diff
@@ -52,6 +54,16 @@ public ProcessInstance createProcessInstance(Process process, CorrelationKey cor
         }
         // TODO: should be part of processInstanceImpl?
         VariableScope variableScope = (VariableScope) ((ContextContainer) process).getDefaultContext( VariableScope.VARIABLE_SCOPE );
+		if (variableScope != null) {
+			for (Variable variable : variableScope.getVariables()) {
+				DataType dataType = variable.getType();
+				Object defaultValue = variable.getMetaData("defaultValue");
+				if (defaultValue != null && defaultValue instanceof String && variable.getValue() == null) {
+					defaultValue = dataType.readValue((String) defaultValue);
+					variable.setValue(defaultValue);
```

Comment:
the variable scope is the "model" ... yo need to set the "instance"

if(varaibleScopeInstance.getVariable() == null) {
   varaibleScopeInstance.setVariable(name, defaultValue)
}